@fabaff PTAL at aiopvpc tests.

```
_________________ test_number_of_national_holidays[2021-104-7] _________________

year = 2021, days_weekend_p3 = 104, extra_days_p3 = 7

    @pytest.mark.parametrize(
        "year, days_weekend_p3, extra_days_p3",
        (
            (2021, 104, 7),
            (2022, 105, 7),
            (2023, 105, 9),
        ),
    )
    def test_number_of_national_holidays(year, days_weekend_p3, extra_days_p3):
        """Calculate days with full P3 valley period."""
        holidays_p3 = weekend_days_p3 = 0
        day = datetime(year, 1, 1, 15, tzinfo=REFERENCE_TZ)
        while day.year == year:
            period, _, _ = get_current_and_next_tariff_periods(day, False)
            if period == "P3":
                if day.isoweekday() > 5:
                    weekend_days_p3 += 1
                else:
                    holidays_p3 += 1
            day += timedelta(days=1)
        assert weekend_days_p3 == days_weekend_p3
>       assert holidays_p3 == extra_days_p3
E       assert 8 == 7
E         +8
E         -7

tests/test_tariff_attributes.py:34: AssertionError
_________________ test_number_of_national_holidays[2022-105-7] _________________

year = 2022, days_weekend_p3 = 105, extra_days_p3 = 7

    @pytest.mark.parametrize(
        "year, days_weekend_p3, extra_days_p3",
        (
            (2021, 104, 7),
            (2022, 105, 7),
            (2023, 105, 9),
        ),
    )
    def test_number_of_national_holidays(year, days_weekend_p3, extra_days_p3):
        """Calculate days with full P3 valley period."""
        holidays_p3 = weekend_days_p3 = 0
        day = datetime(year, 1, 1, 15, tzinfo=REFERENCE_TZ)
        while day.year == year:
            period, _, _ = get_current_and_next_tariff_periods(day, False)
            if period == "P3":
                if day.isoweekday() > 5:
                    weekend_days_p3 += 1
                else:
                    holidays_p3 += 1
            day += timedelta(days=1)
        assert weekend_days_p3 == days_weekend_p3
>       assert holidays_p3 == extra_days_p3
E       assert 8 == 7
E         +8
E         -7

tests/test_tariff_attributes.py:34: AssertionError
_________________ test_number_of_national_holidays[2023-105-9] _________________

year = 2023, days_weekend_p3 = 105, extra_days_p3 = 9

    @pytest.mark.parametrize(
        "year, days_weekend_p3, extra_days_p3",
        (
            (2021, 104, 7),
            (2022, 105, 7),
            (2023, 105, 9),
        ),
    )
    def test_number_of_national_holidays(year, days_weekend_p3, extra_days_p3):
        """Calculate days with full P3 valley period."""
        holidays_p3 = weekend_days_p3 = 0
        day = datetime(year, 1, 1, 15, tzinfo=REFERENCE_TZ)
        while day.year == year:
            period, _, _ = get_current_and_next_tariff_periods(day, False)
            if period == "P3":
                if day.isoweekday() > 5:
                    weekend_days_p3 += 1
                else:
                    holidays_p3 += 1
            day += timedelta(days=1)
        assert weekend_days_p3 == days_weekend_p3
>       assert holidays_p3 == extra_days_p3
E       assert 10 == 9
E         +10
E         -9

tests/test_tariff_attributes.py:34: AssertionError
=========================== short test summary info ============================
FAILED tests/test_tariff_attributes.py::test_number_of_national_holidays[2021-104-7]
FAILED tests/test_tariff_attributes.py::test_number_of_national_holidays[2022-105-7]
FAILED tests/test_tariff_attributes.py::test_number_of_national_holidays[2023-105-9]
================== 3 failed, 39 passed, 4 deselected in 2.81s ==================
builder for '/nix/store/qaglwg1v7z29j22241pk87qld28hfn1i-python3.9-aiopvpc-3.0.0.drv' failed with exit code 1
```

--- a/pkgs/development/python-modules/aiodiscover/default.nix
+++ b/pkgs/development/python-modules/aiodiscover/default.nix
@@ -12,7 +12,7 @@
 
 buildPythonPackage rec {
   pname = "aiodiscover";
-  version = "1.4.7";
+  version = "1.4.8";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
@@ -21,7 +21,7 @@ buildPythonPackage rec {
     owner = "bdraco";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-NtiShZpPFl+elYNPLaKAg6uV8pDJv0pyR+NTUiFoMm0=";
+    sha256 = "sha256-ts3PnL1ePWJHHaLuC0eQKqBOt+HeEeAayeVBAIhnUWs=";
   };
 
   propagatedBuildInputs = [

--- a/pkgs/development/python-modules/aiohomekit/default.nix
+++ b/pkgs/development/python-modules/aiohomekit/default.nix
@@ -12,7 +12,7 @@
 
 buildPythonPackage rec {
   pname = "aiohomekit";
-  version = "0.6.11";
+  version = "0.7.15";
   format = "pyproject";
 
   disabled = pythonOlder "3.7";
@@ -21,7 +21,7 @@ buildPythonPackage rec {
     owner = "Jc2k";
     repo = pname;
     rev = version;
-    sha256 = "1rrdzzb2gcl3lc8l5vb99hy2lmdj5723fds2q78n4sf83y93czw7";
+    sha256 = "sha256-na94H1ll9aZz/mQ4/y3b2vhSaOgo4OjtIsAfW6zGYyI=";
   };
 
   nativeBuildInputs = [
@@ -33,6 +33,8 @@ buildPythonPackage rec {
     cryptography
     zeroconf
   ];
+
+  doCheck = lib.versionAtLeast pytest-aiohttp.version "1.0.0";
 
   checkInputs = [
     pytest-aiohttp

--- a/pkgs/development/python-modules/holidays/default.nix
+++ b/pkgs/development/python-modules/holidays/default.nix
@@ -11,14 +11,14 @@
 
 buildPythonPackage rec {
   pname = "holidays";
-  version = "0.12";
+  version = "0.13";
   format = "setuptools";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "d99f2b6ddc5bfab7b7f8bbed457a82104f8980122a04b982bfc0e4f8820a1d46";
+    sha256 = "sha256-xvfDq4ralIBnAtqTHZTTfNYbz6kstNOdNRtqnFIQZ1w=";
   };
 
   propagatedBuildInputs = [

